matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -1301,6 +1301,21 @@ internal class DefaultCryptoService @Inject constructor(
         cryptoStore.logDbUsageInfo()
     }
 
+    override fun ensureOutboundSession(roomId: String) {
+        cryptoCoroutineScope.launch(coroutineDispatchers.crypto) {
+            roomEncryptorsStore
+                    .get(roomId)
+                    ?.let {
+                        getEncryptionAlgorithm(roomId)?.let { safeAlgorithm ->
+                            val userIds = getRoomUserIds(roomId)
+                            if (setEncryptionInRoom(roomId, safeAlgorithm, false, userIds)) {
+                                roomEncryptorsStore.get(roomId)?.ensureOutboundSession(getRoomUserIds(roomId))
```

Comment:
I think we miss the notify withHeld session by just doing this.
Wouldn't it be better to call alg.encryptEventContent with a sort of dummy param? to avoid code duplication

Reply:
You are right, good idea. Done.